README.md

```diff
@@ -66,6 +66,21 @@ describe('Google', () => {
 })
 ```
 
+If you are using `react-scripts`, you will need to include the following above imports:
+
+```js
+/**
+ * @jest-environment puppeteer
+ */
+```
+
+or alternatively pass the env via command line:
+
+```js
+  "web:test": "react-app-rewired test --env=puppeteer",
+```
+
+
```

Comment:
I think the command line should be standard way so people don't need to add the comment on every test file.

```suggestion
If you are using `react-scripts`, you will need to pass the environment via command line:

```js
  "test": "react-scripts test --env=puppeteer",
```

or alternatively include the following comment at the top of each test file:

```js
/**
 * @jest-environment puppeteer
 */
```

```